recipe/build-r-base.sh

```diff
@@ -265,7 +265,9 @@ Mingw_w64_makefiles() {
     sed -i.bak 's|-lf77blas -latlas|-llapack -lblas|g' src/extra/blas/Makefile.win
     rm src/extra/blas/Makefile.win.bak
     echo "MULTI =   "                                   >> "${SRC_DIR}/src/gnuwin32/MkRules.local"
-    echo "BUILD_HTML = YES"                             >> "${SRC_DIR}/src/gnuwin32/MkRules.local"
+    # BUILD_HTML causes filenames with special characters to be created, see
+    #   https://github.com/conda-forge/r-base-feedstock/pull/177#issuecomment-845279175
+    # echo "BUILD_HTML = YES"                            >> "${SRC_DIR}/src/gnuwin32/MkRules.local"
```

Comment:
1. Does anyone feel this is a big regression for our package? Non-Windows platform did not carry those HTML files at all.
2. Does someone know if there is an upstream issue for having filenames with unescaped special characters?
   ref: https://github.com/conda-forge/r-base-feedstock/pull/177#issuecomment-845349047

Reply:
No issues from me, upstream should really address this.

Reply:
> 1. Does anyone feel this is a big regression for our package? Non-Windows platform did not carry those HTML files at all.

I don't *think* so, but I'm not 100% sure. https://cran.r-project.org/doc/manuals/r-release/R-admin.html#Help-options says "By default HTML help pages are created when needed rather than being built at install time.", which seems like a reasonably good idea to me.

> 2. Does someone know if there is an upstream issue for having filenames with unescaped special characters?
>    ref: [#177 (comment)](https://github.com/conda-forge/r-base-feedstock/pull/177#issuecomment-845349047)

I don't see anything, e.g., at https://bugs.r-project.org/bugzilla/buglist.cgi?quicksearch=html%20help